src/main/java/data/UserRepository.java

```diff
@@ -25,15 +25,15 @@ public UserRepository() {
                 Title.MS,
                 "Maria",
                 "Kvitka",
-                "maria.kvitka3@test.com",
-                "test123",
+                System.getProperty ( "mariaEmail" ),
+                System.getProperty ( "masterPass" ),
                 "03/03/2001",
                 Boolean.TRUE,
                 Boolean.TRUE);
 
         admin = new User();
-        admin.setEmail("admin@gmail.com");
-        admin.setPassword("admin");
+        admin.setEmail(System.getProperty ( "adminEmail" ));
```

Comment:
move all getProperty to helper class
make sure tests fail gracefully if property if not set
extract all required properties in beforeSuite
throw new SkipException("Required property is missing" )  if it is not set

Reply:
it wasn`t my task)we will correct it)